contrail-ui/src/components/main/mainframe/MainFrame.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import Container from '@material-ui/core/Container';
+import AppBar from '@material-ui/core/AppBar';
+import Drawer from '@material-ui/core/Drawer';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import List from '@material-ui/core/List';
+import Box from '@material-ui/core/Box';
+import Paper from '@material-ui/core/Paper';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import FileCopyIcon from '@material-ui/icons/FileCopy';
+import GroupIcon from '@material-ui/icons/Group';
+import SetttingsIcon from '@material-ui/icons/Settings';
+import { withStyles } from "@material-ui/styles";
+import theme from '../../../theme';
+import styles from "../../auth/styles";
```

Comment:
Why are you using my styles lol

You should copy the styles from your template and put it in a new file, and then import that one instead